fsd_utils/config/commonconfig.py

```diff
@@ -282,6 +278,40 @@ class CommonConfig:
             "section_weighting": None,
         },
     )
+
+    # ---------------
+    #  Fund Config
+    # ---------------
+
+    COF_FUND_ID = "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4"
+    COF_ROUND_2_ID = "c603d114-5364-4474-a0c4-c41cbf4d3bbd"
+    COF_ROUND_2_W3_ID = "5cf439bf-ef6f-431e-92c5-a1d90a4dd32f"
+    COF_R2_W3_DEFAULT_LAUNCH_TIME = "2023-02-08 12:00:00"
+    DEFAULT_FUND_ID = COF_FUND_ID
+
+    @classmethod
+    def get_default_round_id(cls):
+        launch_time = os.getenv(
+            "COF_R2_W3_LAUNCH_TIME", cls.COF_R2_W3_DEFAULT_LAUNCH_TIME
+        )
+        print(f"COF R2W3 launch time from env: {str(launch_time)}")
```

Comment:
Can we remove this print, or do a proper formatted log line using the logger?

Reply:
Still a print now due to the time in the startup process when this is called